app/bundles/LeadBundle/Tests/Controller/Api/LeadApiControllerFunctionalTest.php

```diff
@@ -198,7 +198,7 @@ public function testBachdDncAddAndRemove()
         $clientResponse = $this->client->getResponse();
         $response       = json_decode($clientResponse->getContent(), true);
 
-        $this->assertFalse(isset($response['contacts'][0]['doNotContact'][0]));
+        $this->assertEmpty($response['contacts'][0]['doNotContact']);
```

Comment:
This was minor change preset in my previous PRs. I'm only using simpler syntax. I can remove this.

I did it because this test was failing even the code was correct.

Reply:
> I can remove this

I think we can keep this. I was curious If you made these changes intentionally.

Reply:
Kept